### DISCUSSION TITLE
What is the reason for `screen.widgets` removed?

### DISCUSSION THREAD

**Discussion by tsingwang** [Textualize/textual#1858] [Q&A] [answered]
In my opinion, `screen.widgets` is useful. Example:
```py
"""Refresh widgets in current screen only."""
for widget in self.screen.widgets:
    widget.do_custom_refresh()
```
At this point there is no need to focus on background screen widgets.

**Comment by willmcgugan** [comment, marked as answer]
You can do `for widget in self.screen.query("*")` to get all widgets.
